interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -391,6 +391,18 @@ public void largeUnary() throws Exception {
    * a message's compression level, so this is primarily intended to run against a gRPC C++ server.
    */
   public void clientCompressedUnary() throws Exception {
+    clientCompressedUnary(true);
+  }
+
+  /**
+   * Tests client per-message compression for unary calls. The Java API does not support inspecting
+   * a message's compression level, so this is primarily intended to run against a gRPC C++ server.
+   */
+  public void clientCompressedUnaryNoProbe() throws Exception {
```

Comment:
Didn't catch this earlier - checkstyle fails here due to the separation between `clientCompressedUnary()` and `clientCompressedUnary(boolean)`. I'm ok with just dropping the no-arg methods (`clientCompressedUnary()` and `clientCompressedUnaryNoProbe()`) and directly invoking `clientCompressedUnary(true or false)` in `TestServiceClient`. Same for clientCompressedStreaming